packages/medusa/src/services/__tests__/line-item.js

```diff
@@ -407,6 +406,7 @@ describe("LineItemService", () => {
           metadata: {},
           should_merge: true,
           includes_tax: true,
+          order_edit_id: null,
```

Comment:
todo: this field should not appear here as the flag is not enabled for the order edit feature in that describe.

Reply:
you can see that in the other pr we did not had to do that. what happened in your cases that you needed to add it?